### DISCUSSION TITLE
Broken Hyperlink in React Foundations Course

### DISCUSSION THREAD

**Discussion by ItsANoBrainer** [vercel/community#5613] [Bug Report]
### Page to Investigate

https://nextjs.org/learn/react-foundations/server-and-client-components#summary

### Steps to Reproduce

The URL in [The "use client" Directive](https://react.dev/reference/react/use-client%3E) hyperlink has a greater than sign at the end which causes the URL to be invalid and leads to nothing. Removing the greater than symbol fixes this.

**Comment by leerob** [comment]
Thank you! Fixing this.
